system/jlib/jlog.cpp

```diff
@@ -2851,6 +2873,7 @@ class jlib_decl CComponentLogFileCreator : implements IComponentLogFileCreator,
     StringBuffer logDirSubdir;
 
     bool         rolling;
+    long         maxLogFileSize;
```

Comment:
Could initialize the value here.  I've been doing that for all new member variables.

Reply:
I've done this.  Note that I have had to move the variable to make it the last member variable, otherwise the compiler complains.